src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -66,6 +66,19 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
   const { pageInfo } = artist.auctionResultsConnection ?? {}
   const { hasNextPage, endCursor } = pageInfo ?? {}
   const artistName = artist.name
+  const queryString = require("query-string")
+
+  // scroll to auction results if param flag is present
+  useEffect(() => {
+    const queryParams = queryString.parse(location.search)
+    if (queryParams.scroll_to_auction_results) {
+      scrollIntoView({
+        selector: "#scrollTo--artistAuctionResultsTop",
+        behavior: "smooth",
+        offset: 150,
+      })
+    }
+  }, [queryString])
```

Comment:
Maybe we should run this effect only if `scroll_to_auction_results` changes in the URL. Right now we would scroll to the "Auction Results" section any time `scroll_to_auction_results` and any query param changes (this happens every time we change a filter on the page).